inc/shortcodes/glossary/class-glossary.php

```diff
@@ -294,7 +292,7 @@ function tooltipContent( $content ) {
 
 		foreach ( $glossary_terms as $glossary_term_id => $glossary_term ) {
 			$identifier = 'term_' . $id . '_' . $glossary_term_id;
-			$content .= '<template id="' . $identifier . '"><div class="glossary__definition" role="dialog" data-id="' . $identifier . '"><div tabindex="-1">' . wpautop( do_shortcode( $glossary_term ) ) . '</div><button><span aria-hidden="true">&times;</span><span class="screen-reader-text">' . __( 'Close definition', 'pressbooks' ) . '</span></button></div></template>';
+			$content .= '<template id="' . $identifier . '"><div class="glossary__definition" role="dialog" data-id="' . $identifier . '"><div tabindex="-1">' . wpautop( apply_shortcodes( $glossary_term ) ) . '</div><button><span aria-hidden="true">&times;</span><span class="screen-reader-text">' . __( 'Close definition', 'pressbooks' ) . '</span></button></div></template>';
```

Comment:
This is just an alias. I just replaced it, to do a tiny step towards: https://github.com/pressbooks/pressbooks/issues/2183